en/micro-integrator/mkdocs.yml

```diff
@@ -51,6 +51,7 @@ nav:
       - 'Quick Start Guide': 'overview/quick-start-guide.md'
       - 'Developing your First Integration': 'develop/integration-development-kickstart.md'
       - 'Key Concepts': 'overview/key-concepts.md'
+      - 'About this Release': 'overview/about-this-release-7.1.0.md'
```

Comment:
The release number should not be in the name of the .md file. It will not be reusable for future releases if so.